frontend/lib/package.json

```diff
@@ -60,7 +61,7 @@
     "d3-graphviz": "^2.6.1",
     "decamelize": "^6.0.0",
     "deck.gl": "^8.9.36",
-    "dompurify": "^3.1.3",
+    "dompurify": "^3.0.9",
```

Comment:
nit: this might need to be changed back to `3.1.3` (from a recent security dependabot update)